lib/chef/knife/bmcs_server_show.rb

```diff
@@ -4,6 +4,21 @@
 require 'chef/knife/bmcs_common_options'
 require 'chef/knife/bmcs_helper'
 
+# Methods to extend the instance model
+module ServerDetails
```

Comment:
It should not be hard to do that, but I wanted to get this working well first (also, the multi-vnic output patches are dependent on this one).